dali/pipeline/executor/executor_impl.cc

```diff
@@ -495,10 +495,6 @@ void Executor<WorkspacePolicy, QueuePolicy>::RunHelper(OpNode &op_node, Workspac
       DALI_ENFORCE(
           static_cast<size_t>(ws.NumOutput()) == output_desc.size(),
           "Operator::Setup returned shape and type information for mismatched number of outputs");
-      DALI_ENFORCE(op.CanInferOutputs(),
```

Comment:
Should we add a sanity check after the Run to verify if indeed the operator that claims `HasContiguousOutputs` didn't break the promise and cause the internal error?

Reply:
Well..... that's something where a build with asserts would be nice. `IsContiguousInMemory` isn't exactly free and we use it anyway when we _need_ the memory to be contiguous.
I'll add an `assert`.